src/Compilers/Core/Portable/PEWriter/MetadataWriter.PortablePdb.cs

```diff
@@ -847,6 +847,7 @@ private void EmbedCompilationOptions(CommonPEModuleBuilder module)
             WriteValue(CompilationOptionNames.CompilerVersion, compilerVersion);
 
             WriteValue(CompilationOptionNames.Language, module.CommonCompilation.Options.Language);
+            WriteValue(CompilationOptionNames.SourceFileCount, module.CommonCompilation.SyntaxTrees.Count().ToString());
```

Comment:
>Count() [](start = 100, length = 7)

Minor thing and not related to your change but just noticed that SyntaxTrees is IEnumerable<SyntaxTree> but both C# and VB represent it as ImmutableArray<SyntaxTree>. So we are boxing it unnecessarily. Can't change public API but could change and use CommonSyntaxTrees.

Reply:
I think that's a good idea. Let's spin it off into its own issue.